### PR TITLE
Eng 872 bumped dependencies

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-composite-app-controller).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-composite-app-controller/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 95bfb0b397e24056bc137a1cb06d0117521bdb65 | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-composite-app-controller/PR-11/1.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-composite-app-controller-carcpr11**

File: .idea/runConfigurations/Tags__in_process__.xml
Comment (joewhite101) [inline .idea/runConfigurations/Tags__in_process__.xml:1]: Is this checked in on purpose? Should .idea be ignored

**Review by joewhite101 — CHANGES_REQUESTED** [review]
Question on IDE file.

**Review by joewhite101 — APPROVED** [review]
LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-composite-app-controller/pull/11#pullrequestreview-510266168" title="Approved">joewhite101</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-composite-app-controller).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-k8s-composite-app-controller/blob/master/OWNERS)~~ [joewhite101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: .idea/runConfigurations/Tags__in_process__.xml
Comment (joewhite101) [reply .idea/runConfigurations/Tags__in_process__.xml:1]: File is to support pre loaded config for E2E tests

Comment (entando-jx) [reply .idea/runConfigurations/Tags__in_process__.xml:1]: It was intentional to make the runConfigurations available to other developers

**Pull request merged**
